tests/cephadm/test_remove_service.py

```diff
@@ -0,0 +1,54 @@
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class RemoveService(Exception):
+    pass
+
+
+def _verify_service(node, service_name):
+    """
+    Verifies service is removed
+    Args:
+        node (str): monitor node object
+        service_name (str): service name
+    return: (bool)
+    """
+    # Get service status
+    out = CephAdm(node).ceph.orch.ls(service_type=service_name, format="json-pretty")
+    # If service was removed, wait for a timeout to check whether its removed
+    timeout, interval = 20, 2
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        out = CephAdm(node).ceph.orch.ls(
+            service_type=service_name, format="json-pretty"
+        )
+        if "No services reported" in out:
+            return True
+    if w.expired:
+        log.info(f"Service {service_name} is not removed after rm operation.")
```

Comment:
Instead of adding log here and raising exeption later, I suggest to raise exception here only.

Reply:
Updated